Sources/Penny/+String.swift

```diff
@@ -2,8 +2,22 @@ import Foundation
 import Models
 
 extension String {
```

Comment:
I get why this is duplicated here, but it would be very preferable for it not to be. (I realize that implies adding a separate utilities target to the package, though; I don't consider it a blocker for this PR.)

Reply:
yeah but i don't want to make a whole new target just because of 1 extension 😅 maybe when there are a few of them

Reply:
Like I said, not a blocker here 🙂